care.js

```diff
@@ -115,21 +118,35 @@ function doTheTweets() {
 function doTheCodes() {
   var todayCommits = 0;
   var weekCommits = 0;
-
-  var today = spawn('sh ' + __dirname + '/standup-helper.sh', ['-m ' + config.depth, config.repos], {shell:true});
   todayBox.content = '';
-  today.stdout.on('data', data => {
-    todayCommits = getCommits(`${data}`, todayBox);
-    updateCommitsGraph(todayCommits, weekCommits);
-    screen.render();
-  });
-
-  var week = spawn('sh ' + __dirname + '/standup-helper.sh', ['-m ' + config.depth + ' -d 7', config.repos], {shell:true});
   weekBox.content = '';
-  week.stdout.on('data', data => {
-    weekCommits = getCommits(`${data}`, weekBox);
-    updateCommitsGraph(todayCommits, weekCommits);
+
+  function getCommits(data, box) {
+    var commitRegex = /(.......) (- .*)/g;
+    var content = colorizeLog(data);
+    box.content += content;
+    return (box.content.match(commitRegex) || []).length;
+  }
+
+  function showError(err, box) {
+    getCommits(`😥  ${err}`, box);
```

Comment:
Is this bit actually needed? For an error, can't we just do `box.content = err`?